site/en/guides/border-router/_toc.yaml

```diff
@@ -1,70 +1,60 @@
 toc:
 - title: Overview
   path: /guides/border-router/
-- title: Get Started
-  path: /guides/border-router/get-started
 
 - heading: Platforms
 - title: Android
   path: /android/build-an-android-border-router
-- title: BeagleBone Black (Linux)
-  path: /guides/border-router/beaglebone-black
 - title: Raspberry Pi (Linux)
-  path: /guides/border-router/raspberry-pi
+  path: /guides/border-router/overview
 - title: Espressif ESP32 Series (FreeRTOS)
   path: /guides/border-router/espressif-esp32
 
 - heading: Setup
-- title: Build and Configuration
-  path: /guides/border-router/build
+- title: Overview
+  path: /guides/border-router/overview
+  step_group: otbr_setup
+- title: Setup (Docker)
+  path: /guides/border-router/build-docker
   step_group: otbr_setup
-- title: Web GUI
-  path: /guides/border-router/web-gui
+- title: Setup (Native)
+  path: /guides/border-router/build-native
+  step_group: otbr_setup
+- title: Form Thread Network
+  path: /guides/border-router/form-network
   step_group: otbr_setup
 - title: mDNS Discovery
   path: /guides/border-router/mdns-discovery
   step_group: otbr_setup
 
-- title: External Commissioning
+- heading: Codelabs
+- title: OpenThread Border Router
+  status: external
+  path: /codelabs/openthread-border-router/
+- title: OpenThread Border Router IPv6 Multicast
+  status: external
+  path: /codelabs/openthread-border-router-ipv6-multicast
+- title: OpenThread Border Router NAT64
+  status: external
+  path: /codelabs/openthread-border-router-nat64
+
+- heading: External Commissioning
```

Comment:
You can't do heading with section, this needs to be `title: External Commissioning`

However, is there a need to have this collapsed at this point?  Can we go without a section?

Reply:
I made an update here to have:
```
- heading: Thread Commissioning
- title: External Commissioning
  section:
  - title: Overview
    path: /guides/border-router/external-commissioning
    step_group: external_commissioning
...
```
Does this work?